README.md

```diff
@@ -215,6 +215,28 @@ fmt.Println(issues, githubError, resp, err)
 
 Pass a nil `successV` or `failureV` argument to skip JSON decoding into that value.
 
+### HTTP Tracing
+
+Go 1.7+ supports [HTTP tracing](https://blog.golang.org/http-tracing).  Trace with a Sling:
+
+```go
+issues := new([]Issue)
+githubError := new(GithubError)
+req, _ := githubBase.New().Get(path).QueryStruct(params).Request()
```

Comment:
Can you make this just a plain example request rather than associated with the Github example?

```
req, err := sling.New().Get("https://example.com").QueryStruct(params).Request()
...

client.Do(req)
```